docs/install_guide.adoc

```diff
@@ -175,6 +175,86 @@ $ oc describe secret osp-secret -n openstack | grep Watcher
 WatcherPassword:                  9 bytes
 ----
 +
+. Watcher does not support creating certificates for TLS, but assumes that the
```

Comment:
At the beginning of this section, I'd make more explicit that these certificates are required to expose the Watcher services with https in both public and internal endpoints.

Reply:
I've tried to make the sentence describing the certificates more explicit